src/Chirp.Infrastructure/UserData.cs

```diff
@@ -0,0 +1,5 @@
+using Chirp.Core;
+
+public class UserData {
```

Comment:
should contain some attributes but not an author object, otherwise it loses purpose and an author in auth.csthml.cs would suffice

Reply:
id, name, email